modules/olm-restricted-networks-configuring-operatorhub.adoc

```diff
@@ -112,8 +122,48 @@ command and can be used to further customize the mirroring configuration.
 . If you used the `--manifests-only` flag in the previous step and want to mirror
 only a subset of the content:
 
-.. Modify the list of images in your `mapping.txt` file to your
-specifications.
+.. Modify the list of images in your `mapping.txt` file to your specifications. If
+you are unsure of the exact names and versions of the subset of images you want
+to mirror:
+
+... Run the `sqlite3` tool against the temporary database that was generated by the
+`oc adm catalog mirror` command to retrieve a list of images matching a general
+search query.
++
+For example, to retrieve a list of images that that are similar to the string
+`clusterlogging.4.3`:
++
+[source,terminal]
+----
+$ echo "select * from related_image \
+    where operatorbundle_name like 'clusterlogging.4.3.%';" \
+    | sqlite3 -line /tmp/190214037/bundles.db <1>
+----
+<1> Refer to the previous output of the `oc adm catalog mirror` command to find the
+path of the database file.
++
+.Example output
+[source,terminal]
+----
+image = registry.redhat.io/openshift4/ose-logging-kibana5@sha256:aa4a8b2a00836d0e28aa6497ad90a3c116f135f382d8211e3c55f34fb36dfe61
+operatorbundle_name = clusterlogging.4.3.33-202008111029.p0
+
+image = registry.redhat.io/openshift4/ose-oauth-proxy@sha256:6b4db07f6e6c962fc96473d86c44532c93b146bbefe311d0c348117bf759c506
+operatorbundle_name = clusterlogging.4.3.33-202008111029.p0
+[...]
+----
+
+... Use the results from the previous step to edit the `mapping.txt` file to only
+include the subset of images you want to mirror. For example, you can conclude
```

Comment:
Slightly confused about this sentence. You're concluding that they exist, but were you not the one to add it, as of the first sentence in this step? Is it actually trying to say that these are the example lines that you would add to your mapping.txt file, according to the output from the previous step? Or that there are already entries in there, and you would remove all the others except these two lines?

Reply:
The latter.

Reply:
* In the previous "Run the sqlite tool" step, added "The output helps inform how you will later edit your `mapping.txt` file.
* Changed the "you can conclude line" here to "For example, you can use the `image` values from previous example output to find that the following matching lines exist in your `mapping.txt` file:"
* Added after the example `mapping.txt` image mappings: "In this example, if you only wanted to mirror these images, you would then
remove all other entries in the `mapping.txt` file and leave only the above two
lines."